packages/core/src/components/dialog/_multistep-dialog.scss

```diff
@@ -0,0 +1,87 @@
+// Copyright 2020 Palantir Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0.
+
+@import "~@blueprintjs/icons/src/icons";
+@import "../../common/mixins";
+@import "../../common/react-transition";
+@import "../../common/variables";
+
+$dialog-border-radius: $pt-border-radius * 2 !default;
+$step-radius: $pt-border-radius * 2 !default;
+
+.#{$ns}-multistep-dialog-panels {
+  display: flex;
+}
+
+.#{$ns}-multistep-dialog-left-panel {
+  border-radius: 0 $dialog-border-radius 0 0;
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+}
+
+.#{$ns}-multistep-dialog-right-panel {
+  background-color: $light-gray5;
+  border-left: 1px solid $pt-divider-black;
+  border-radius: 0 0 $dialog-border-radius 0;
+  flex: 2;
+}
+
+.#{$ns}-multistep-dialog-footer {
+  background-color: $white;
+  border-radius: 0  0 $dialog-border-radius 0;
+  border-top: 1px solid $pt-divider-black;
+  padding: 10px;
+}
+
+.#{$ns}-step-container {
+  background-color: $light-gray5;
+  border-bottom: 1px solid $pt-divider-black;
+
+  &.#{$ns}-active {
+    background-color: $white;
+  }
+}
+
+.#{$ns}-step {
+  align-items: center;
+  background-color: $light-gray5;
+  border-radius: $step-radius;
+  cursor: not-allowed;
+  display: flex;
+  margin: 4px;
+  padding: 6px;
```

Comment:
Nit: why not just 10px of padding? I think would expect these to highlight full-bleed like rows in a table.

Reply:
I think Carol's original designs wanted only a part of the button to be highlighted when we hover over it. The margin enables us to prevent that space from being highlighted as well. 
 
![image](https://user-images.githubusercontent.com/36139290/103228800-a199be80-48ff-11eb-93d4-dbf29dffc26e.png)

Not sure if this is the still the case, but if not, I can change it to 10px padding.